src/dataviews/category-dataview-model.js

```diff
@@ -152,7 +151,7 @@ module.exports = DataviewModelBase.extend({
   },
 
   getCount: function () {
-    return this.get('allCategoryNames').length;
+    return this.get('categoriesCount');
```

Comment:
And why this change says that it was using 'allCategoryNames' size?
I'm confused.